pages/linux/viewnior.md

```diff
@@ -0,0 +1,15 @@
+# viewnior
+
+> Simple and elegant image viewer.
+
+- View an image:
+
+`viewnior {{path/to/image.png}}`
```

Comment:
Does viewnior only support png images?

Reply:
Nope! It supports any kind of image you can think of. I'll update this now.